src/components/landing_page/LandingPage.scss

```diff
@@ -0,0 +1,41 @@
+.LandingPage {
+  width: 100%;
+  height: calc(100% - var(--footer-height));
+}
+
+.Content {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-family: 'Source Sans Pro', sans-serif;
+  font-weight: 200;
+
+  h1 {
+    color: var(--accent-color-blue);
+    font-size: 4em;
+  }
+
+  .subtitle {

```

Comment:
Erik wants us to use less of the CSS selector syntax and use explicit classes instead.
Aka `&__I_love_underscores`. I think he has some good reasons for that but I'll let him explain that himself.

Reply:
Very true!
Please do it something like this:
```
.LandingPage {
  &__title {
      ...
   }

   &__subtitle { 
     ....
   }

  &__content {
    
  }
}
```
So your class names have all to be something like `LandingPage__content`, `LandingPage__title`.
By this we prevent conflicts with classes used multiple times in different contexts resulting in unexpected changes everywhere in the app.

Reply:
You can read more about this [here](http://getbem.com/naming/).
This is a methodology called BEM.